### PR TITLE
improvement: add local dev path

### PR DESCRIPTION
**Pull request by khaya-zulu**

### CODE REVIEW

**Review by peterp — APPROVED** [review]

File: docs/06-tutorials/supabase-clone-environments.md [hunk 1]
```diff
@@ -98,24 +106,7 @@ If you want to learn more about Snaplet, you can explore our docs. If you have a
...
-supabase init
-
-# With docker installed and running start up your supabase services locally
-supabase start
-
-# Grab the local database url and insert into the ./snaplet/config.json file
-# Restore to the target database url
-snaplet ss r --no-reset
-```
+You can use the `generate` command to seed your local dev setup. Read this [guide](https://supabase.com/docs/guides/cli/seeding-your-database) for more information.
```
Comment (justinvdm) [inline docs/06-tutorials/supabase-clone-environments.md]: The docs currently read like one would always first restore, even if they want to run `generate`. It is more that the user would _either_ restore data, or generate data.

* If restoring: They would restore using a snapshot from production, at which point they are good to go
* If generating: They would run their migration tool, and then afterwards run `snaplet generate` - i.e. no restoring involved

Should we maybe make the docs read this way? e.g. Have step 6 be "restore _or_ generate"

Comment (khaya-zulu) [reply docs/06-tutorials/supabase-clone-environments.md]: That's a good point, I added the `generate` command as an exploration (You can also do seeding in your database). But I like the idea of splitting the guide into these two branches.

Comment (khaya-zulu) [reply docs/06-tutorials/supabase-clone-environments.md]: @justinvdm, if it's okay with you I made this [update](https://github.com/snaplet/docs/pull/179/files#diff-3de484223bc5343190e0172d6cfe898aae7899a9a1cb132bafad136036064642R111). Just so we centering this guide only around restoring.

Comment (justinvdm) [reply docs/06-tutorials/supabase-clone-environments.md]: Looks great, thanks Khaya

**Pull request merged**